src/domains/chart/components/lib-charts/dygraph-chart.tsx

```diff
@@ -456,6 +466,27 @@ export const DygraphChart = ({
           updateChartPanOrZoom({ after: minDate, before: maxDate })
         },
 
+        underlayCallback(canvas: CanvasRenderingContext2D, area: DygraphArea, g: Dygraph) {
+          // the chart is about to be drawn
+          // this function renders global highlighted time-frame
+
+          if (propsRef.current.globalChartUnderlay) {
+            const { after, before } = propsRef.current.globalChartUnderlay
+
+            if (after < before) {
+              const bottomLeft = g.toDomCoords(after, -20)
```

Comment:
What is this `20`?

Reply:
That was copy/pasted code from different place, but overall good point, i fixed it! 
 But i left the `const HIGHLIGHT_HORIZONTAL_PADDING = ` right above that code, so it's easy to red, if that's ok